beacon-chain/slasher/chunks.go

```diff
@@ -492,12 +516,12 @@ func (m *MinSpanChunksSlice) NextChunkStartEpoch(startEpoch primitives.Epoch) pr
 //	max_spans_val_i = [[-, -, -], [-, -, -], [-, -, -]]
 //
 // If C = chunkSize is 3 epochs per chunk, and we input start epoch of chunk 1 which is 3. The next start
-// epoch is the start epoch of chunk 2, which is epoch 4. This is computed as:
+// epoch is the start epoch of chunk 2, which is epoch 6. This is computed as:
 //
 //	first_epoch(chunkIndex(startEpoch)+1)
 //	first_epoch(chunkIndex(3)+1)
 //	first_epoch(1 + 1)
-//	first_epoch(2)
+//	first_epoch(6)
 //	4
```

Comment:
```suggestion
//	first_epoch(2)
//	6
```
I think this is what the author meant. The last value is the epoch. I read this as `first_epoch(chunk_2) => 6`

Reply:
I don't know how I ended up with this conclusion 😂